gpu/segmentation/include/pcl/gpu/segmentation/gpu_extract_labeled_clusters.h

```diff
@@ -39,121 +39,173 @@
 
 #pragma once
 
-#include <pcl/point_types.h>
-#include <pcl/point_cloud.h>
+#include <pcl/gpu/octree/octree.hpp>
 #include <pcl/PointIndices.h>
 #include <pcl/pcl_macros.h>
-#include <pcl/gpu/octree/octree.hpp>
+#include <pcl/point_cloud.h>
+#include <pcl/point_types.h>
 
-namespace pcl
-{
-  namespace gpu
+namespace pcl {
+namespace gpu {
+template <typename PointT>
+void
+extractLabeledEuclideanClusters(
+    const typename pcl::PointCloud<PointT>::Ptr& host_cloud_,
+    const pcl::gpu::Octree::Ptr& tree,
+    float tolerance,
+    std::vector<PointIndices>& clusters,
+    unsigned int min_pts_per_cluster,
+    unsigned int max_pts_per_cluster);
+
+/** \brief @b EuclideanLabeledClusterExtraction represents a segmentation class for
+ * cluster extraction in an Euclidean sense, depending on pcl::gpu::octree \author Koen
```

Comment:
Doxygen seems to be problematic in a lot of places

Reply:
Thanks a lot, Kunal, for the review! Oh boy, clang-format and Doxygen seem to get along with each other as well as cats and dogs do. I tried to make sure that clang-format places the \param keyword on a new line but did not realize that other keywords get misrepresented. I will change that!